docs/config-file/v2.rst

```diff
@@ -26,15 +26,18 @@ Below is an example YAML file which shows the most common configuration options:
            os: ubuntu-20.04
            tools:
              python: "3.9"
-             nodejs: "16"
+             # You can also specify other tool versions:
+             # nodejs: "16"
+             # rust: "1.55"
+             # golang: "1.17"
 
          # Build documentation in the docs/ directory with Sphinx
          sphinx:
             configuration: docs/conf.py
 
          # Optionally build your docs in additional formats such as PDF
-         formats:
-            - pdf
+         # formats:
+            # - pdf
```

Comment:
```suggestion
         # formats:
         #    - pdf
```

Reply:
I would align it with the previous line's comment as well 😅

Reply:
Sorry, I enabled the automerge and didn't see this. Is it just a style preference? I checked the current version and it works